scripts/create_person.rb

```diff
@@ -0,0 +1,8 @@
+caren = License.create surname: 'Bishop',
+                      given_name: 'Caren',
+                      gender: 'f',
+                      height: 63,
+                      weight: 132,
+                      born_on: '1943-09-26',
+                      eye_color: 'Brown',
+                      hair_color: 'Black'
```

Comment:
Correct (though since your migration didn't create any columns for your table, the code will error out).